Source/CombatExtended/CombatExtended/Loadouts/Dialog_ManageLoadouts.cs

```diff
@@ -25,12 +25,12 @@ public class Dialog_ManageLoadouts : Window
         #region Fields
 
         private static Texture2D
-            _arrowBottom = ContentFinder<Texture2D>.Get("UI/Icons/arrowBottom"),
-            _arrowDown = ContentFinder<Texture2D>.Get("UI/Icons/arrowDown"),
-            _arrowTop = ContentFinder<Texture2D>.Get("UI/Icons/arrowTop"),
-            _arrowUp = ContentFinder<Texture2D>.Get("UI/Icons/arrowUp"),
+            //_arrowBottom = ContentFinder<Texture2D>.Get("UI/Icons/arrowBottom"),
```

Comment:
If the code works without those, we should probably remove them and not comment them out.

Reply:
Usually with unused variables, I assume someone might have been planning to use them, so prefer to leave them, just commented out, as that makes it easier for whomever had plans for them later.  Of course, I think we've had complete turnover in devs, so whomever may have planned to use them is reasonably unlikely to care.  I can make another commit.